README.md

```diff
@@ -110,11 +114,32 @@ Jump the current song to a time
 
 - milliseconds `Number` - Integer representing milliseconds to jump the current track to
 
+#### `playback.getSongInfo()`
```

Comment:
We should provide some description for the method (even though it's relatively obvious). How about:

```
#### `playback.getSongInfo()`
Retrieve current song's metadata
```